src_main/xevdm_ipred.c

```diff
@@ -303,6 +303,18 @@ void xevdm_ipred_uv(pel *src_le, pel *src_up, pel *src_ri, u16 avail_lr, pel *ds
     }
 }
 
+int intra_mode_list[IPD_CNT] = {
```

Comment:
can you add "static" in front of this variable? if it is used only for this code file.

Reply:
I add "static".